doc/source/structures/vessels/dockingport.rst

```diff
@@ -5,12 +5,6 @@ DockingPort
 
 Some of the Parts returned by :ref:`LIST PARTS <list command>` will be of type :struct:`DockingPort`.
 
-.. note::
-
-    .. versionadded:: 0.18
-        The spelling of suffixes `AQUIRERANGE`, `AQUIREFORCE`, and `AQUIRETORQURE` on the :struct:`DockingPort` structure has been corrected.  Please use `ACQUIRERANGE`, `ACQUIREFORCE`, and `ACQUIRETORQURE` instead.  Using the old incorrect spelling, a deprecation exception will be thrown, with instruction to use the new spelling.
-
-
```

Comment:
Disagree.  Old scripts might not spell them the new way.

Reply:
And this one *has* the deprecation exception.  I thought I saw this somewhere.